sites/diverseeducation.com/server/templates/static-pages/mppwcc-yearly.marko

```diff
@@ -302,6 +301,77 @@ $ const cheeGoals = currentYearData ? currentYearData.cheeGoals: title;
           </table>
         </@section>
       </if>
+$ const tableFourLabels = Object.values(getAsObject(currentYearData, "tableLabels.tableFour"));
+      <if(currentYearData && tableFourLabels.length)>
+        <@section modifiers=["card-deck"]>
+          <div class="row">
+            <div class="col">
+              <h2 class="page-wrapper__website-section-name">ENROLLMENT TYPE</h2><br/>
+            </div>
+          </div>
+          <table class="table table-responsive table-hover">
+            <thead>
+              <tr>
+                <td><b>Institution</b></td>
+                <for|label| of=tableFourLabels>
+                  <td>
+                    <b>${label}</b>
+                  </td>
+                </for>
+              </tr>
+            </thead>
```

Comment:
Try experimenting with width attributes on the td's here, I believe ```sites/diverseeducation.com/server/templates/static-pages/mppwsa-yearly``` did something similar in order to get the tables to fill out the pages better as here is what I'm getting as of time of comment:


<img width="1792" alt="Screen Shot 2022-05-25 at 10 53 14 AM" src="https://user-images.githubusercontent.com/46794001/170306846-6cc86342-3eae-4ddd-a745-453a72c59f54.png">
<img width="1792" alt="Screen Shot 2022-05-25 at 10 53 20 AM" src="https://user-images.githubusercontent.com/46794001/170306855-d71f4050-d8a3-47e5-ba5b-7a107d651746.png">

Reply:
@flaviogalizia See the previous tables in this file